hw-6/microsoft/css/style.scss

```diff
@@ -0,0 +1,340 @@
+/* Переменные */
+
+$grey: #262626;
+$background: #F3F3F3;
+
+/* Шаблоны */
+
+%padding {
+    padding: 0 64px;  
```

Comment:
what is this???????

Reply:
Dodislav Petrovich its bokovoy otstup kotoriy na sayte.